From c64e2a7932aae81b40507f94a44c73b172cc2688 Mon Sep 17 00:00:00 2001
From: harvey186 <harvey186@hotmail.com>
Date: Mon, 4 Aug 2025 18:51:34 +0100
Subject: [PATCH 1/1] Remove calendar tests

We remove the calendar app itself so the tests are pointless

Change-Id: If42d07277984643c6fadafe55703a95217f455c6
---
 Android.bp | 1 -
 1 file changed, 1 deletion(-)

diff --git a/Android.bp b/Android.bp
index 009f390..9a05995 100644
--- a/Android.bp
+++ b/Android.bp
@@ -24,7 +24,6 @@ test_package {
         "benchmarks",
         "BootHelperApp",
         "BusinessCard",
-        "CalendarTests",
         "CellBroadcastReceiverTests",
         "ConnectivityUIDTest",
         "CtsCameraTestCases",
--
2.49.0
